client/repo.c

```diff
@@ -467,6 +467,155 @@ TDNFEventRepoMDDownloadEnd(
     goto cleanup;
 }
 
+uint32_t
+TDNFStoreBaseURLFromMetalink(
+    PTDNF pTdnf,
+    const char *pszRepo,
+    const char *pszRepoMDURL
+    )
+{
+    uint32_t dwError = 0;
+    char *pszBaseUrlFile = NULL;
+    PTDNF_REPO_DATA_INTERNAL pRepos = NULL;
+
+    if (!pTdnf ||
+       !pTdnf->pConf ||
+       IsNullOrEmptyString(pszRepo) ||
+       IsNullOrEmptyString(pszRepoMDURL))
```

Comment:
These lines have to move forward by one space char. Align folds **after** the opening '(' in IF.